specs/governance/gov-token.md

```diff
@@ -24,74 +24,42 @@
+`GovernanceToken` MUST have a `mint(address,uint256)` function with external visibility that allows the `owner()` address
 to mint an arbitrary number of new tokens to a specific address. This function MUST only be called by the contract
-owner, the `MintManager`, as enforced by the `onlyOwner` modifier inherited from the `Ownable` contract. When tokens
-are minted, the voting power of the recipient address MUST be updated accordingly in the `Alligator` contract via the
-`afterTokenTransfer` hook. The total token supply is capped to `2^208^ - 1` to prevent overflow risks in the voting
-system. If the total supply exceeds this limit, `_mint(address,uint256)`, as inherited from `ERC20Votes`, MUST revert.
+owner, the `MintManager`, as enforced by the `onlyOwner` modifier inherited from the `Ownable` contract.
 
 ### Token Burning
 
 The contract MUST allow token holders to burn their own tokens using the inherited `burn(uint256)` or
-`burnFrom(address,uint256)` functions inherited from `ERC20Burnable`. When tokens are burned, the total supply and the
-holder's voting power MUST be reduced accordingly in the `Alligator` contract via the `afterTokenTransfer` hook.
+`burnFrom(address,uint256)` functions inherited from `ERC20Burnable`. Burn functions MUST NOT allow a user's balance to
+underflow and attempts to reduce balance below zero MUST revert.
 
 ### Voting Power
 
-Each token corresponds to one unit of voting power.
-By default, token balance does not account for voting power. To have their voting power counted, token holders MUST delegate
-their voting power to an address (can be their own address).
-The contract MUST offer public accessors for querying voting power, as outlined below.
+Each token corresponds to one unit of voting power. Token holders who wish to utilize their tokens for voting MUST delegate
+their voting power to an address (can be their own address). An active delegation MUST NOT prevent a user from transferring
+tokens. The `GovernanceToken` contract MUST offer public accessor functions for querying voting power, as outlined below.
 
-#### Queries
+#### Public Query Functions
 
-- The `getVotes()(uint256)` function MUST retrieve the current voting power of an address from the `Alligator` contract.
-- The `getPastVotes(address,uint256)(uint256)` function MUST allow querying the voting power of an address at a specific
-  block number in the past from the `Alligator` contract.
-- The `getPastTotalSupply(uint256)(uint256)` function MUST return the total voting power at a specific block number in
-  the past from the `Alligator` contract.
+- `getVotes() returns (uint256)`
+  - MUST retrieve the current voting power of an address.
+- `getPastVotes(address,uint256) returns (uint256)`
+  - MUST retrieve the voting power of an address at specific block number in the past.
+- `getPastTotalSupply(uint256) returns (uint256)`
+  - MUST return the total token supply at a specific block number in the past.
```

Comment:
we are missing a few so that it matches what we specify in alligator as well (https://github.com/ethereum-optimism/specs/pull/247/files#diff-30586f0d9290fe2d91fb11d4a4a8676629526de3269ef327f449ef555993274dR16-R21):
- `checkpoints`
- `numCheckpoints`
- `delegates`

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.9.6/contracts/token/ERC20/extensions/ERC20Votes.sol#L57-L111

Reply:
applies to the other govtoken doc too